settings.gradle

```diff
@@ -2,5 +2,6 @@ rootProject.name = 'embedded-elasticsearch'
 
 include 'core'
 include 'test-base'
+include 'es1x-test'
```

Comment:
I would follow convention and name this module "es17-text", because you are testing that this tool works with 1.7 version not 1.x in general. We may have situation where in some minor version, for example 5.1 we may need introduce some version specific changes, so then we will have es17-test, es24-test, es50-test, es51-test, etc